client/js/components/procedure/publicindex/DpSearchProcedureMap.vue

```diff
@@ -328,6 +328,26 @@ export default {
   },
 
   methods: {
+    fitToBounds () {
+      setTimeout(() => {
+        if (window.markersLayer.getLayers().length > 0) {
+          /*
+           * {
+           *   # Here, the
+           *   fitBounds
+           *   #
```

Comment:
what is this? Do you still need it?

Reply:
its just moved code without touching. Can be deleted. I don't really care.
```suggestion
```